examples/js/e-commerce/index.html

```diff
@@ -25,7 +25,7 @@
     <link rel="stylesheet" href="./src/app.css" />
     <link rel="stylesheet" href="./src/app.mobile.css" />
     <link rel="stylesheet" href="./src/widgets/PriceSlider.css" />
-    <script src="https://polyfill.io/v3/polyfill.min.js?features=default%2CArray.prototype.find%2CArray.prototype.includes%2CPromise%2CObject.assign%2CObject.entries"></script>
+    <script src="../../polyfills.js"></script>
```

Comment:
wouldn't it be better if the file was local so it won't error on codesandbox?

Reply:
Oh I forgot they are also on CSB. I'll see if I can just symlink the same source file or if I just have to hard copy it.

Reply:
hard copy is probably easiest. Not sure if symlink will work in csb. This file shouldn't change anyway